inbm-lib/inbm_lib/dbs_parser.py

```diff
@@ -26,8 +26,8 @@ def parse_docker_bench_security_results(dbs_output: str) -> Dict[str, Union[bool
     fails = "Failures in: "
     success_flag = True
     prev_warn = False
-    failed_images: List = []
-    failed_containers: List = []
+    failed_images: list[str] = []
```

Comment:
These will be removed with my PR.  Created a dataclass instead.  Could we remove this change to save the merge conflict?

Reply:
@nmgaston I can wait until your PR is merged in and then sort it out here so once this PR is in, all of inbm-lib will pass the strict checks.